xml/deployment_installation.xml

```diff
@@ -825,9 +825,253 @@
+      <procedure>
+
+      <step>
+       <para>
+        Go to <guimenu>Project -> Compute -> Images</guimenu> and click on 
+        <guimenu>Create Image</guimenu>.
+       </para>
+       <para>
+        Give your image a name (for example: <literal>CaaSP-3</literal>) you will
+        need to use this later to find the image.
+       </para>
+      </step>
+       <step>
+        <para>
+         Go to <guimenu>Project -> Orchestration</guimenu> and click on 
+         <guimenu>Stacks</guimenu>.
+        </para>
+        <informalfigure>
+         <mediaobject>
+          <imageobject role="fo">
+           <imagedata fileref="horizon_stacks.png" width="100%"/>
+          </imageobject>
+          <imageobject role="html">
+           <imagedata fileref="horizon_stacks.png" width="100%"/>
+          </imageobject>
+         </mediaobject>
+        </informalfigure>
+       </step>
+       
+       <step>
+        <para>
+         Click on <guimenu>Launch Stack</guimenu>.
+        </para>
+       </step>
+       
+       <step>
+        <para>
+         You need to provide the stack templates at this stage.
+         Either upload the files, provide the URL to the raw files directly 
+         (only applies to stack template) or copy and paste the contents into the 
+         <literal>Direct Input</literal> fields.
+        </para>
+        <warning>
+         <title>Replace the default <replaceable>root_password</replaceable></title>
+          <para>
+           Do not use the <filename>caasp-environment.yaml</filename> via it's URL
+           directly from the GitHub repository.
+          </para>
+           <para>
+            You must make sure to replace the value for 
+            <replaceable>root_password</replaceable> with a secure password.
+            This will become the password for the root account on all nodes in the
+            stack.
+           </para>
+         <screen>
+&prompt.user;<command>cat caasp-environment.yaml</command>
+---
+parameters:
+  root_password: <replaceable>linux</replaceable>
+  admin_flavor: m1.large
+...
+         </screen>
+       </warning>
+
+        <informalfigure>
+         <mediaobject>
+          <imageobject role="fo">
+           <imagedata fileref="horizon_launch_stack.png" width="100%"/>
+          </imageobject>
+          <imageobject role="html">
+           <imagedata fileref="horizon_launch_stack.png" width="100%"/>
+          </imageobject>
+         </mediaobject>
+        </informalfigure>
+       </step>
+       
+      <step>
+       <para>
+        Click <guimenu>Next</guimenu>.
+       </para>
+      </step>
+
+ <!-- FIXME mnapp, 17/6/18 This screenshot has a keepassxc icon in the root password box, replace -->
+       <step>
+        <para>
+         Now you need to define more information about the stack.
+        </para>
+        <itemizedlist>
+         <listitem><para>Stack Name - Give your stack a name</para></listitem>
```

Comment:
listitem and para are block elements, not inline elements. So you should insert line breaks and indent the text accordingly.